android/Omnivore/app/src/main/res/xml/data_extraction_rules.xml

```diff
@@ -5,15 +5,17 @@
 -->
 <data-extraction-rules>
     <cloud-backup>
-        <!-- Use <include> and <exclude> to control what is backed up.
-        <include .../>
-        <exclude .../>
-        -->
+        <exclude domain="root" />
```

Comment:
Sorry what is this stuff for?

Reply:
It disables [Android Auto Backup](https://developer.android.com/identity/data/autobackup) which uploads the app data like SQLite and datastore data which includes the auth token. Uploading SQLite data isn't necessary and might even cause problems because the data might have been updated after the backup. And uploading the auth token is obviously bad.

Reply:
oh awesome, thanks!